packages/opentelemetry-plugin-xml-http-request/src/xhr.ts

```diff
@@ -316,6 +316,7 @@ export class XMLHttpRequestPlugin extends BasePlugin<XMLHttpRequest> {
     }
 
     const currentSpan = this._tracer.startSpan(url, {
+      kind: api.SpanKind.CLIENT,
```

Comment:
It would be nice if you can add check in the unit test.

Reply:
@mayurkale22 test added

Reply:
Thank you :)